src/components/CalendarDay.jsx

```diff
@@ -334,6 +340,18 @@ export default withStyles(({ reactDates: { color, font } }) => ({
     },
   },
 
+  CalendarDay__before_today: {
+    background: color.before_today.backgroundColor,
```

Comment:
this seems like something that should be applied as an empty object, so that you can override it with a theme - rather than hardcoding something here.

Reply:
perhaps it would also be cleaner to simply always include the class, regardless?

Reply:
That's what i'm thinking - to always include the class, but by default have that provide no styling.

That way, you can override it without needing to alter the component API.